WBooks/Library/LibraryViewController.swift

```diff
@@ -67,7 +66,7 @@ class LibraryViewController: UIViewController {
                                   NSLayoutConstraint(item: headerImage, attribute: .top, relatedBy: .equal,
```

Comment:
Also, there is a much nicer way of creating constraint from iOS 9 onwards, with anchors:
```
NSLayoutConstraint.activate([
    headerImage.centerXAnchor.equal(to: self.view.centerXAnchor),
    ...other constraint...,
    ...another constraint...
])
```